python/ray/log_monitor.py

```diff
@@ -100,6 +100,9 @@ def close_all_files(self):
                 if (file_info.worker_pid != "raylet"
                         and file_info.worker_pid != "gcs_server"
                         and file_info.worker_pid != "autoscaler"):
+                    assert not isinstance(file_info.worker_pid, str), (
```

Comment:
lol imo `assert type(file_info.worker_pid) == str` is a little easier to read, but i'll minimize my nitpicking :p